_example/db/db.go

```diff
@@ -17,10 +17,13 @@ import (
 func Connect() *gorm.DB {
 	dir := filepath.Dir("db/database.db")
 	db, err := gorm.Open("sqlite3", dir+"/database.db")
+
```

Comment:
This empty line is unnecessary. AFAIK most of Go code do not insert empty line between getting error and judging whether the error is nil.

It is probably clear where the error is derived from.

```go
db, err := gorm.Open("sqlite3", dir+"/database.db")
if err != nil {
// ...
}
```